charts/hono/templates/hono-adapter-amqp/hono-adapter-amqp-vertx-secret.yaml

```diff
@@ -41,4 +41,5 @@ data:
   cert.pem: {{ .Files.Get "example/certs/amqp-adapter-cert.pem" | b64enc }}
   trusted-certs.pem: {{ .Files.Get "example/certs/trusted-certs.pem" | b64enc }}
   adapter.credentials: {{ .Files.Get "example/amqp-adapter.credentials" | b64enc }}
-{{- end }}
\ No newline at end of file
+  truststore.jks: {{ .Files.Get "example/certs/trustStore.jks" | b64enc }}
```

Comment:
what do we need this for? We already configure the *trusted-certs.pem* which contains all the trusted certificates ....

Reply:
Kafka does not support _pem_ files before version 2.7. Hono uses the Kafka clients in version 2.6.

Reply:
But Hono also supports JKS files so FMPOV we should either use JKS *or* PEM files, but not both ...

Reply:
I can provide a PR that replaces the `trusted-certs.pem` with a JKS file if you like.

Reply:
with *a JKS* file? My understanding is that we should be able to use *only* the `truststore.jks` file or does that contain other certs than `trusted-certs.pem`?

Reply:
Right, `trustStore.jks` is the JKS file to be used everywhere instead of `trusted-certs.pem`.

Reply:
I think you should indeed change it that way.

Reply:
@sophokles73 Please have a look at #257.

Reply:
> Kafka does not support pem files before version 2.7. Hono uses the Kafka clients in version 2.6.

Is there a particular reason why we do not simply use the 2.7.x client and continue to use our PEM files?

Reply:
> > Kafka does not support pem files before version 2.7. Hono uses the Kafka clients in version 2.6.
> 
> Is there a particular reason why we do not simply use the 2.7.x client and continue to use our PEM files?

Hono does not directly use the Kafka clients, but they are a transitive dependency of the Vert.x Kafka client. Currently, it is version `3.9.7` of `vertx-kafka-client` which depends on version `2.6.0` of the Kafka clients.

Another minor point might be client/broker compatibility. Kafka version 2.7 is pretty recent. While I find it hard to find general reliable information on that topic, it seems advisable to use brokers with an equal or higher version than the clients. So, this would potentially increase the minimal Kafka version required by Hono.